### ISSUE TITLE
Method: locations.fetchMultiDailyMetricsTimeSeries missing!

### ISSUE THREAD

**Issue by gellezzz** [googleapis/google-api-php-client-services#2440]
#### Environment details

  - OS: Amazon Linux AMI 2018.03
  - PHP version: 7.2.34
  - Package name and version: google/apiclient-services: v0.293.0

#### Steps to reproduce

  From documentation [https://developers.google.com/my-business/reference/performance/rest/v1/locations](https://developers.google.com/my-business/reference/performance/rest/v1/locations) only this getDailyMetricsTimeSeries method exists. I can't find fetchMultiDailyMetricsTimeSeries...
I checked in the Node.js and Go libraries where the method exists, but not in the PHP library.
 

#### Code example

```php
$gmb = new BusinessProfilePerformance($gClient);
$gmb->locations->getDailyMetricsTimeSeries($name, $optParams) //exists
$gmb->locations->fetchMultiDailyMetricsTimeSeries(...) //not exists
```

**Comment by ajupazhamayil** [comment]
We have merged the changes required for this function to main.

Ref: https://github.com/googleapis/google-api-php-client-services/blob/main/src/BusinessProfilePerformance.php#L67

Will be available in the next release.

Thank you for raising this issue.

**Comment by ajupazhamayil** [comment]
Released v0.294.0

Closing this issue, thank you!
